tests/module_test.go

```diff
@@ -0,0 +1,67 @@
+package testing
+
+import (
+	"io/ioutil"
+	"log"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+)
+
+func TestModule(t *testing.T) {
+	files, err := ioutil.ReadDir("./")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	for _, f := range files {
+		// look for directories with test cases in it
+		if f.IsDir() {
```

Comment:
ought to be able to combine the two tests to collapse the nesting...
```suggestion
		if f.IsDir() && f.Name() != "vendor" {
```